Cargo.toml

```diff
@@ -75,12 +77,12 @@ codegen-units = 1
 version_check = "0.9.4"
 
 [dependencies]
-const-random = { version = "0.1.12", optional = true }
+const-random = { version = "0.1.17", optional = true }
 serde = { version = "1.0.117", optional = true }
 cfg-if = "1.0"
 atomic-polyfill = { version="1.0.1", optional=true}
 getrandom = { version = "0.2.7", optional = true }
-zerocopy = { version = "0.7.14", default-features = false, features = ["simd"] }
+zerocopy = { version = "0.7.20", default-features = false, features = ["simd"] }
```

Comment:
Raising minimum version in semver is not necessary unless your crate fails to build / support lower versions.

There is `-Z direct-minimal-versions` which (_unlike `-Z minimal-versions_) will resolve only the lowest bound of your semver range for your dependencies (_`-Z minimal-versions` does this for every crate in `Cargo.lock` which is more likely to fail_). Using this would then verify a build compatibility at your lower bound, while the implicit dependencies would resolve their semver as per usual.

Raising the minimum version adds a constraint on the semver if other crates have an overlapping range, narrowing what is accepted AFAIK.

If your declared `rust-version` builds / resolves correctly, then there is no need to raise the minimum? It does not benefit your crate when published AFAIK, semver will be resolved when `ahash` is pulled in and typically use the latest release for the dependency unless `Cargo.lock` for example manipulates that like with the mentioned `-Z` nightly options.